proxmoxtf/provider/schema.go

```diff
@@ -98,5 +98,35 @@ func nestedProviderSchema() map[string]*schema.Schema {
 			},
 			ValidateFunc: validation.StringIsNotEmpty,
 		},
+		mkProviderSSHUsername: {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Used to override the ssh username",
+			DefaultFunc: schema.MultiEnvDefaultFunc(
+				[]string{"PROXMOX_VE_SSHUSERNAME", "PM_VE_SSHUSERNAME"},
+				nil,
+			),
+			AtLeastOneOf: []string{
```

Comment:
That was a simple error, I was ignorant of what the option does. After reading up on the docs it's clear that it doesn't belong at all since it'll copy username if it's not set anyways.